src/hoi4_world/localizations/localizations_converter_tests.cpp

```diff
@@ -0,0 +1,102 @@
+#include <sstream>
+
+#include "external/commonItems/external/googletest/googlemock/include/gmock/gmock-matchers.h"
+#include "external/commonItems/external/googletest/googletest/include/gtest/gtest.h"
+#include "src/hoi4_world/localizations/localizations_converter.h"
+
+
+
+namespace hoi4
+{
+
+TEST(Hoi4worldLocalizationsLocalizationsconverter, NoCountryMappingsNoCountryLocalizations)
+{
+   commonItems::LocalizationDatabase vic3_localizations("english", {"spanish"});
+   commonItems::LocalizationBlock block_one("VIC", "english");
+   block_one.ModifyLocalization("english", "test");
+   block_one.ModifyLocalization("spanish", "prueba");
+   vic3_localizations.AddOrModifyLocalizationBlock("VIC", block_one);
+   commonItems::LocalizationBlock block_two("TWO", "english");
+   block_two.ModifyLocalization("english", "test two");
+   block_two.ModifyLocalization("spanish", "prueba dos");
+   vic3_localizations.AddOrModifyLocalizationBlock("TWO", block_two);
+
+
+   const Localizations hoi4_localizations = ConvertLocalizations(vic3_localizations, {});
+
+   EXPECT_EQ(hoi4_localizations.GetCountryLocalizations().size(), 0);
+}
+
+
+TEST(Hoi4worldLocalizationsLocalizationsconverter, NoVic3LocalizationsNoHoi4CountryLocalizations)
```

Comment:
Marginally related, but "Return English, if present, if required language is not found" is a cure-all for all incoming mods that localize only english.

Reply:
Yep! Demonstrated by the output tests.